app/views/templates/search/search_object.rb

```diff
@@ -323,11 +436,12 @@ def provenance_data
       
       def property_data
         
-        properties    = collect_values(['proxies.dctermsExtent'])
-        propertiesCS  = collect_values(['proxies.dcMedium', 'proxies.dcDuration'])
-        propertiesFmt = collect_values(['aggregations.webResources.dcFormat'])
-          
-        props = [].push(*propertiesFmt).push(*properties).push(*propertiesCS)
+        properties_cs  = collect_values(['proxies.dcMedium', 'proxies.dcDuration'])
+        properties_fmt = collect_values(['aggregations.webResources.dcFormat'])
+        properties_xt  = collect_values(['proxies.dctermsExtent'])
+        properties_lng = collect_values(['proxies.dcLanguage'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.